WorkflowFunction/Manager/AuthorizationWorkflowRightManager.php

```diff
@@ -2,6 +2,8 @@
 
 namespace OpenOrchestra\WorkflowFunction\Manager;
 
+use Doctrine\Common\Collections\ArrayCollection;
```

Comment:
you should use `Doctrine\Common\Collections\Collection` instead, it allows to have more collection type (PersistentCollection for instance)